services/motion/builtin/base_scenes_test.go

```diff
@@ -0,0 +1,196 @@
+package builtin
```

Comment:
We probably don't actually want to commit this file

Reply:
Yeah this is largely duplicative of `motion-testing` code